src/js/media-captions-button.js

```diff
@@ -118,6 +119,9 @@ class MediaCaptionsButton extends MediaChromeButton {
 
   constructor(options = {}) {
     super({ slotTemplate, ...options });
+    // Internal variable to keep track of when we have some or no captions (or subtitles, if using subtitles fallback)
+    // Used for `default-showing` behavior.
+    this.__captionsReady = false;
```

Comment:
Nit: We've used single underscore for private vars everywhere else so far. I don't really care one way or another but we should be consistent.

Reply:
nah should be single for consistency.